nnvm/tests/python/frontend/tensorflow/test_forward.py

```diff
@@ -787,29 +789,72 @@ def _get_sample(data, state):
         np.testing.assert_allclose(tf_samples, tvm_samples, rtol=1e-5, atol=1e-5)
         assert(tvm_sample_str == tf_sample_str)
 
-#######################################################################
-# LRN (Local Response Normalization)
-# ----------------------------------
 
-def _test_lrn(ishape, size, axis, bias, alpha, beta):
-    """ testing local response normalization """
-    lrn_depth_radius = size / 2
+#######################################################################
+# Split
+# ------
+def _test_split(ip_shape, num_split, axis):
+    tf.reset_default_graph()
+    dtype = 'float32'
+    in_data = tf.placeholder(dtype, ip_shape, name="in_data")
+    tf.split(in_data, num_split, axis=axis, name="split")
+    np_data = np.random.uniform(size=ip_shape).astype(dtype)
+    with tf.Session() as sess:
+        final_graph_def = tf.graph_util.convert_variables_to_constants(
+            sess,
+            sess.graph.as_graph_def(add_shapes=True),
+            ['split'])
+
+        tf_out_node_names = ['split:%g' % i for i in range(num_split)]
+        tf_output = run_tf_graph(sess, [np_data], ['in_data:0'], tf_out_node_names)
+        tvm_out_shapes = [tf_output[i].shape for i in range(num_split)]
+        tvm_out_dtypes = [tf_output[i].dtype for i in range(num_split)]
+        tvm_output = run_tvm_graph(final_graph_def, [np_data], ['in_data'],
+                                   tvm_out_shapes, tvm_out_dtypes)
+        np.testing.assert_allclose(tf_output, tvm_output, atol=1e-5, rtol=1e-5)
+        sess.close()
 
-    inp_array = np.random.uniform(size=ishape).astype(np.float32)
+def test_forward_split():
+    '''test split operator'''
+    _test_split((2, 3), 2, axis=0)
+    _test_split((6, 3), 3, axis=0)
+    _test_split((5, 9, 3), 3, axis=1)
+    _test_split((2,5,3,9), 3, axis=3)
 
-    with tf.Graph().as_default():
-        in1 = tf.placeholder(shape=inp_array.shape, dtype=inp_array.dtype, name="lrn0_data")
-        nn_ops.local_response_normalization(in1,
-                                            name="lrn",
-                                            depth_radius=lrn_depth_radius,
-                                            bias=bias,
-                                            alpha=alpha,
-                                            beta=beta)
 
-        compare_tf_with_tvm(inp_array, 'lrn0_data:0', 'lrn:0')
+#######################################################################
+# LRN (Local Response Normalization)
+# ----------------------------------
+def _test_lrn(ip_shape, depth_radius=2, alpha=1e-05, beta=0.75, bias=1.0):
```

Comment:
[Ref.](https://www.tensorflow.org/api_docs/python/tf/nn/local_response_normalization)  we don't need to explicitly set the default values. TF can handle them if not set.

Suggest to ignore to allow TF default values play role here.